composer.json

```diff
@@ -4,7 +4,7 @@
   "license": "APACHE2",
   "homepage": "https://pipl.com/api/",
   "require": {
-    "php": ">=5.3.0",
+    "php": ">=7.1.0",
```

Comment:
I think that we need to consult with product about the language level. it may effect customers

Reply:
i spoke about it to roy kfir, 
php has deprecated those versions:

![image](https://github.com/piplcom/piplapis-php/assets/106513176/9c00af32-7b59-4e20-8cc6-eda9a6d9cb12)

php below v8 doesn't get support and security updates

Reply:
:+1: